typescript/token/src/adapters/EvmTokenAdapter.ts

```diff
@@ -0,0 +1,213 @@
+import { BigNumber, PopulatedTransaction, Signer, providers } from 'ethers';
```

Comment:
This is not new, just moved over from the warp UI like we did for the SealevelTokenAdapter